packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -88,7 +88,12 @@ export class TCompForm extends React.Component {
 
 	componentWillReceiveProps(nextProps) {
 		if (this.props.state.get('properties') !== nextProps.state.get('properties')) {
```

Comment:
I would like to suggest this instead 
```suggestion
		const nextProperties = nextProps.state.get('properties', Map());
		if (this.props.state.get('properties') !== nextProperties) {
			this.setState({ properties: nextProperties.toJS() });
		}
```